app/gameengine.py

```diff
@@ -876,6 +876,15 @@ def find_attachment(self, attachment_id):
                     return attachment
         return None
 
+    def find_card_v2(self, card_id):
```

Comment:
did you try just changing find_card and stuff failed?
I'd rather avoid this kind of thing unless absolutely necessary.

Reply:
I tried refactoring `find_card` to include looking at attachment zones as well but the function's usage has been tightly coupled with the concept that it will NOT look at those zones.

An idea is to rename `find_card` to something else then use this `_v2` to be the actual `find_card`.